### ISSUE TITLE
Custom field options for `OrderableModel.ordering`?

### ISSUE THREAD

**Issue by foucdeg** [matthiask/django-admin-ordering#23]
I'd like to change the `verbose_name` of the `ordering` field because our admin site is not in English. Is there any way to do that?

I've tried assigning it directly after model definition:

```python
class MyModel(OrderableModel):
    ...

MyModel.ordering.verbose_name = gettext_lazy("Ordre")
```
but the admin and the migration generation system still consider the verbose_name to be `ordering`.

**Comment by matthiask** [comment]
Hi

The best option would be to contribute a new translation to the project :-)

You'd only have to translate a single string:
https://github.com/matthiask/django-admin-ordering/blob/main/admin_ordering/locale/de/LC_MESSAGES/django.po

You don't have to use the `OrderableModel` by the way, you could use your own model and define your own ordering field, maybe even with a different name (if you set `ordering_field`); but, the best option is to contribute the translation.

**Comment by foucdeg** [comment]
> The best option would be to contribute a new translation to the project :-)

I'll try that, but in case it's faster for you, the word in French is "ordre".
This solution would solve my specific problem but not a more general one which would be to set other model properties, and to set them to different values on different models.

> You don't have to use the OrderableModel by the way.

True. Current solution is to duplicate the field definition from OrderableModel, because I like the other features this model comes with.

**Comment by matthiask** [comment]
No problem. I added the translation myself (but credited you!) and released 0.19.1.

Merci beaucoup et une bonne journée!
